### ISSUE TITLE
Provide a way to use custom payload renderer for Pusher.

### ISSUE THREAD

**Issue by rthideaway** [galaniprojects/liveblog#92]
I was told that this is the main liveblog issue tracker so posting here as well. Details about issue were already described here https://www.drupal.org/node/2862789

**Comment by goldquest** [comment]
Yes, thanks. With this project, we are using a github workflow.
I think it's a good idea and will look into it. Could you make a Pull Request and reference this issue?

**Comment by goldquest** [comment]
Btw, if there are other problems with embedding this into headless sites, please open some issues here.

**Comment by rthideaway** [comment]
PR created. And thanks, I'll come back if something pops up, but for now we're fine, thanks!

**Comment by maosmurf** [comment]
patch from https://www.drupal.org/files/issues/2862789-9.patch is not applicable anymore, will look into this issue for 1.2.0-alpha13 and drupal 8.4

**Comment by maosmurf** [comment]
Patch 2862789-13.patch for alpha13 https://www.drupal.org/files/issues/2862789-13.patch

We still need to find a solution to provide custom renderers, as we push only json to pusher and decoupled frontend takes care of rendering to client.

**Comment by milkovsky** [comment]
Agree, I think it makes also sense to have a pluggable render service.
